node/src/main/kotlin/net/corda/node/services/config/NodeConfiguration.kt

```
@@ -43,7 +43,7 @@ interface NodeConfiguration : NodeSSLConfiguration {
 data class DevModeOptions(val disableCheckpointChecker: Boolean = false)
 
 data class DatabaseConfig(
-        val initDatabase: Boolean = true,
+        val adjustSchemas: Boolean = true,
```

Perhaps annotate this with `@OldConfig` so that the node logs a warning to switch to the new config. Actually I think `OldConfig` should throw an exception rather than logging.

---

Nobody is really using this with `H2` so far, we might still change it in a clean way.

---

We just documented it today. Not released yet.

---

Please do not change the name of this attribute.
`adjustSchemas` is not meaningful.